internal/character/gepard/talent.go

```diff
@@ -0,0 +1,74 @@
+package gepard
+
+import (
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/engine/modifier"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+const (
+	Talent key.Modifier = "gepard-talent"
+)
+
+type talentState struct {
+	revivePerc float64
+	a4Active   bool
+	e6Active   bool
+}
+
+func init() {
+	modifier.Register(Talent, modifier.Config{
+		Listeners: modifier.Listeners{
+			OnLimboWaitHeal: func(mod *modifier.ModifierInstance) bool {
```

Comment:
nit: lets move larger subscription functions to their own method rather than keeping anonymous like this